filehandling.c

```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+
+struct employee
+{
```

Comment:
name of struct, function etcc whould start with Capital letter

struct Employee

Brackets format should be as

struct Employee {
}

If {
}

Reply:
ok